src/plugins/conditionSet/ConditionSetViewProvider.js

```diff
@@ -20,42 +20,50 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-import ConditionSetComponent from './components/ConditionSet.vue';
+import ConditionSet from './components/ConditionSet.vue';
 import Vue from 'vue';
 
-export default function ConditionSet(openmct) {
+export default function ConditionSetViewProvider(openmct) {
     return {
-        key: 'conditionSet',
+        key: 'conditionSet.view',
         name: 'Condition Set',
-        cssClass: 'icon-page',
+        cssClass: 'icon-summary-widget',
         canView: function (domainObject) {
             return domainObject.type === 'conditionSet';
         },
-        view: function (domainObject) {
+        canEdit: function (domainObject) {
+            return domainObject.type === 'conditionSet';
+        },
+        view: function (domainObject, objectPath) {
             let component;
-
             return {
-                show: function (element) {
-                    component =  new Vue({
-                        el: element,
+                show(container) {
+                    component = new Vue({
+                        el: container,
                         components: {
-                            ConditionSetComponent
+                            ConditionSet
                         },
                         provide: {
                             openmct,
-                            domainObject
+                            domainObject,
+                            objectPath
+                        },
+                        data() {
```

Comment:
@JoelMcKinnon This duplicates the data property in the component, i would remove from here